src/Pact/Gas/Table.hs

```diff
@@ -232,21 +238,26 @@ tableGasModel gasConfig =
         GUse _moduleName _mHash -> (_gasCostConfig_useModuleCost gasConfig)
           -- The above seems somewhat suspect (perhaps cost should scale with the module?)
         GInterfaceDecl _interfaceName _iCode -> (_gasCostConfig_interfaceCost gasConfig)
-        GUserApp t -> case t of
-          Defpact -> (_gasCostConfig_defPactCost gasConfig) * _gasCostConfig_functionApplicationCost gasConfig
-          _ -> _gasCostConfig_functionApplicationCost gasConfig
   in GasModel
       { gasModelName = "table"
       , gasModelDesc = "table-based cost model"
       , runGasModel = run
       }
+{-# INLINE tableGasModel #-}
 
 
-perByteFactor :: Decimal
-perByteFactor = 0.1
+perByteFactor :: Rational
+perByteFactor = 1%10
```

Comment:
Should we do something similar with `intialGas` function in Chainweb?
https://github.com/kadena-io/chainweb-node/blob/master/src/Chainweb/Pact/TransactionExec.hs#L716

It's also using Decimal
```
feePerByte :: Decimal = 0.01
```

Reply:
Yes! I'm preparing a PR for this in Chainweb so I'll check that out